### ISSUE TITLE
proxy.golang.org doesn't have v0.21.0 correctly

### ISSUE THREAD

**Issue by kzys** [firecracker-microvm/firecracker-go-sdk#235]
I really don't know why, but it apparently has 7404c2ba5c5bff572d5efb34d9edc3976e70ae56.

```
% curl --silent -LO https://proxy.golang.org/github.com/firecracker-microvm/firecracker-go-sdk/@v/v0.21.0.zip
% unzip -xq v0.21.0.zip
% head github.com/firecracker-microvm/firecracker-go-sdk@v0.21.0/CHANGELOG.md
# 0.20.0
* Moves the NetNS field to `Config` from `JailerConfig` (#155).
* Supports forcing CNI network creation (#130).
* Adds `FIRECRACKER_GO_SDK_INIT_TIMEOUT_SECONDS` and `FIRECRACKER_GO_SDK_REQUEST_TIMEOUT_MILLISECONDS` environment variables to configure timeouts (#165).
* Adds `ForwardSignals` to explicitly configure signal handling (#166).

# 0.19.0
* Firecracker v0.19 API: Vsock API call: PUT /vsocks/{id} changed to PUT /vsock and no longer
  appear to support multiple vsock devices. Any subsequent calls to this API
  endpoint will override the previous vsock device configuration.
%
```

**Comment by stealthybox** [comment]
I think this is related to an issue we had with ignite.
This is how we worked around it:
https://github.com/weaveworks/ignite/commit/19a5e3fec7dc42ad4b215139234f0ee469af2539

**Comment by kzys** [comment]
Resolving since we have released v0.22.0 recently. Please reopen this issue if you must use 0.21.0.
